CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Fixed
+- Use `del-cli` executable instead of `del` in order to not interfere with system commands on Windows ([#7](/../../issues/7))
```

Comment:
please use the full http link (when the package is somewhere installed, the link wouldn't work)